sql/core/src/test/resources/sql-tests/inputs/operators.sql

```diff
@@ -81,3 +81,17 @@ select positive('-1.11'), positive(-1.11), negative('-1.11'), negative(-1.11);
 -- pmod
 select pmod(-7, 2), pmod(0, 2), pmod(7, 0), pmod(7, null), pmod(null, 2), pmod(null, null);
 select pmod(cast(3.13 as decimal), cast(0 as decimal)), pmod(cast(2 as smallint), cast(0 as smallint));
+
+-- width_bucket
+select width_bucket(5.35, 0.024, 10.06, 5);
```

Comment:
Let's add tests for edge cases like value == min, value == max, value < min, value > max, etc.

Reply:
These test cases seems to be covered in the Pg regression tests: https://github.com/apache/spark/pull/28764/files#diff-f97e8257cf76e8ae2bbdd3b79ac926beR820-R830
We need to test them in this suite, too?

Reply:
oh, ok, then it should be fine.

Reply:
Thanks for the check! It seem the coverage of ported Pg tests looks almost perfect.